src/editors/containers/ProblemEditor/components/EditProblemView/index.jsx

```diff
@@ -27,8 +27,10 @@ import messages from './messages';
 import ExplanationWidget from './ExplanationWidget';
 import { saveBlock } from '../../../../hooks';
 
-import { selectors } from '../../../../data/redux';
+import { selectors, actions } from '../../../../data/redux';
+import { getDataFromOlx } from '../../../../data/redux/thunkActions/problem';
```

Comment:
Can we use frontend-aliases for new imports?

Reply:
Currently additional imports removed due to `ProblemEditorPluginSlot` refactored